portal-kernel/src/com/liferay/portal/kernel/portlet/PortletResponseUtil.java

```diff
@@ -265,8 +266,20 @@ protected static void setHeaders(
 			mimeResponse.setContentType(contentType);
 		}
 
-		mimeResponse.setProperty(
-			HttpHeaders.CACHE_CONTROL, HttpHeaders.CACHE_CONTROL_PRIVATE_VALUE);
+		ResourceRequest resourceRequest = (ResourceRequest)portletRequest;
+
+		String resourceID = resourceRequest.getResourceID();
+
+		if (resourceID.equals("kbArticleRSS")) {
+			mimeResponse.setProperty(
+				HttpHeaders.CACHE_CONTROL,
+				HttpHeaders.CACHE_CONTROL_NO_CACHE_VALUE);
+		}
+		else {
+			mimeResponse.setProperty(
+				HttpHeaders.CACHE_CONTROL,
+				HttpHeaders.CACHE_CONTROL_PRIVATE_VALUE);
+		}
```

Comment:
This is not ok. You cannot hardcode KB stuff into the core.

Reply:
If this really needs to be fixed, you should extend the API so that it allows to customize the headers.